### PR TITLE
add A011 checkbox

### PR DESCRIPTION
**Pull request by RichardGarcia**

### CODE REVIEW

File: ui/components/atoms/A011-Checkbox/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import schema from './schema'
+
+const A011 = props => {
+  return (
+    <Checkbox
+      color={props.color}
+      checked={props.checked}
+      disabled={props.disabled}
+      disableRipple={props.disableRipple}
```
Comment (agentlewis) [inline ui/components/atoms/A011-Checkbox/index.js]: we don't need disableRipple as part of our API for this component :)

File: ui/components/atoms/A011-Checkbox/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import createPropTypes from 'json-schema-prop-types'
+
+import Checkbox from '@material-ui/core/Checkbox'
+
+import schema from './schema'
+
+const A011 = props => {
+  return (
+    <Checkbox
+      color={props.color}
```
Comment (agentlewis) [inline ui/components/atoms/A011-Checkbox/index.js]: add the 'id' prop

File: ui/components/atoms/A011-Checkbox/schema.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      type: 'boolean'
+    },
+    color: {
+      type: 'string',
+      enum: ['default', 'primary', 'secondary']
+    },
+    disabled: {
+      type: 'boolean'
+    },
+    disabledRipple: {
```
Comment (agentlewis) [inline ui/components/atoms/A011-Checkbox/schema.js]: remove disabledRipple

File: ui/components/atoms/A011-Checkbox/schema.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    id: {
+      type: 'string'
+    },
+    indeterminate: {
+      type: 'boolean'
+    },
+    value: {
+      type: 'string'
+    },
+    type: {
```
Comment (agentlewis) [inline ui/components/atoms/A011-Checkbox/schema.js]: remove type - we dont need it. or if we do hard code it in the component to 'checkbox'

File: ui/components/atoms/A011-Checkbox/schema.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      type: 'string'
+    }
+  },
+  required: ['value']
```
Comment (agentlewis) [inline ui/components/atoms/A011-Checkbox/schema.js]: Why is value required?

Comment (RichardGarcia) [reply ui/components/atoms/A011-Checkbox/schema.js]: I'm thinking that we need to put a value of each checkbox..

**Pull request merged**